javav2/example_code/medicalimaging/src/main/java/com/example/medicalimaging/UpdateImageSetMetadata.java

```diff
@@ -45,123 +46,155 @@ public static void main(String[] args) {
+                                .build())
+                        .build();
+
+                updateMedicalImageSetMetadata(medicalImagingClient, datastoreId, imagesetId,
+                        versionid, metadataInsertUpdates, force);
+                // snippet-end:[medicalimaging.java2.update_image_set_metadata.insert_or_update_attributes]
+            } else if (updateType.equals("remove_attribute")) {
+                // Remove an attribute.
+                // snippet-start:[medicalimaging.java2.update_image_set_metadata.remove_attributes]
+                final String removeAttributes = """
+                        {
+                          "SchemaVersion": 1.1,
+                          "Study": {
+                            "DICOM": {
+                              "StudyDescription": "CT CHEST"
                             }
                           }
                         }
-                      }
-                    }      
-                    """;
-            MetadataUpdates metadataRemoveUpdates = MetadataUpdates.builder()
-                    .dicomUpdates(DICOMUpdates.builder()
-                            .removableAttributes(SdkBytes.fromByteBuffer(
-                                    ByteBuffer.wrap(removeInstance
-                                            .getBytes(StandardCharsets.UTF_8))))
-                            .build())
-                    .build();
-
-            updateMedicalImageSetMetadata(medicalImagingClient, datastoreId, imagesetId,
-                    versionid, metadataRemoveUpdates);
-            // snippet-end:[medicalimaging.java2.update_image_set_metadata.remove_instance]
+                        """;
+                MetadataUpdates metadataRemoveUpdates = MetadataUpdates.builder()
+                        .dicomUpdates(DICOMUpdates.builder()
+                                .removableAttributes(SdkBytes.fromByteBuffer(
+                                        ByteBuffer.wrap(removeAttributes
+                                                .getBytes(StandardCharsets.UTF_8))))
+                                .build())
+                        .build();
+
+                updateMedicalImageSetMetadata(medicalImagingClient, datastoreId, imagesetId,
+                        versionid, metadataRemoveUpdates, force);
+                // snippet-end:[medicalimaging.java2.update_image_set_metadata.remove_attributes]
+            } else if (updateType.equals("remove_instance")) {
+                // Remove an instance.
+                // snippet-start:[medicalimaging.java2.update_image_set_metadata.remove_instance]
+                final String removeInstance = """
+                        {
+                          "SchemaVersion": 1.1,
+                          "Study": {
+                            "Series": {
+                              "1.1.1.1.1.1.12345.123456789012.123.12345678901234.1": {
```

Comment:
Will these strings run without modification?

Reply:
No, but the complexity of this example does not allow passing arguments. I am prioritizing the service writers' requests. These examples go much more in depth with options than most of our examples.